cernopendata/modules/fixtures/data/docs/cms-about/cms-about.md

```diff
@@ -40,6 +40,7 @@ The following are provided through this portal:
     * It may have double-counting (i.e. a physics object may appear as a single object of its own type, but it may also be part of a jet).
     * Additional knowledge is needed to define a "good" physics object.
     * Definition of same objects is different in each analysis.
+* Some datasets, such as those containing heavy-ion data, are provided in a format called RECO, which contains more information than the AOD format. This is done when the original data analyses by the CMS collaboration used this format.    
```

Comment:
You can say, "This is done when the original analyses by the CMS collaboration was performed using this particular format."

Reply:
@katilp, I realised later (on my phone) that there was a typo. It said "was performed" instead of "were performed". I've now fixed it above.